modules/datastore/docs/query.json

```diff
@@ -193,7 +193,12 @@
                         "-",
                         "*",
                         "/",
-                        "%"
+                        "%",
```

Comment:
@dharizza in the rest of the schema the convention has been to use lowercase, let's do that here and convert to upper in one of the factories if necessary.